examples/browser-test-vite/test/start.js

```diff
@@ -21,11 +23,24 @@ async function startViteServer(opts) {
   };
 }
 
-new BrowserTestDriver().run({
-  title: 'Unit Tests',
-  server: {
-    port: 'auto',
-    start: startViteServer
-  },
-  headless: mode === 'headless'
-});
+async function runTest(testDriver) {
```

Comment:
Seems like passing in the `testDriver` to the `onFinish` function would allow us to avoid this wrapper function?